src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1013,7 +1013,7 @@ public String getGiverNameForResponse(FeedbackResponseAttributes response) {
     public String appendTeamNameToName(String name, String teamName) {
         String outputName;
         if (name.contains("Anonymous") || name.equals(Const.USER_UNKNOWN_TEXT)
-         || name.equals(Const.USER_NOBODY_TEXT) || teamName.isEmpty()) {
+             || name.equals(Const.USER_NOBODY_TEXT) || teamName.isEmpty()) {
```

Comment:
Yup. When it's aligned it can easily be misread as part of the if block construct. In fact Google has another item in their style guide that says the indentation here must be at least 8 spaces (or, if you would, 4 spaces since their indentation length is 2).